rsts/getting_started.sh.rst

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+:'
+
+.. _gettingstarted:
+
+Getting started
+---------------
+
+.. rubric:: Estimated time to complete: 3 minutes.
+
+
+Prerequisites
+***************
+
+Make sure you have `docker installed <https://docs.docker.com/get-docker/>`__ and `git <https://git-scm.com/>`__ installed, then install flytekit:
+
+Steps
+*****
+
+'
+
+# #. First install the python Flytekit SDK (maybe in a virtual environment) and clone the `flytekit-python-template <https://github.com/flyteorg/flytekit-python-template>`_ repo::
+
+  pip install flytekit
+  git clone git@github.com:flyteorg/flytekit-python-template.git myflyteapp
+  cd myflyteapp
+
+
+' #. The repo comes with a sample workflow, which can be found under ``myapp/workflows/example.py``. The structure below shows the most important files and how a typical flyteapp should be laid out.
+.. note::
+
+    .
+    ├── Dockerfile
+    ├── docker_build_and_tag.sh
+    ├── myapp
+    │   ├── __init__.py
+    │   └── workflows
+    │       ├── __init__.py
+    │       └── example.py
+    └── requirements.txt
+
```

Comment:
May be we can ask the user to also see this view using tree command available for his platform.
I couldn't find docker_build_and_tag.sh when i cloned
```
tree myflyteapp/
myflyteapp/
├── Dockerfile
├── LICENSE
├── Makefile
├── README.md
├── boilerplate
│   ├── flyte
│   │   ├── Readme.rst
│   │   ├── config.yml
│   │   └── update.sh
│   ├── update.cfg
│   └── update.sh
├── flyte.config
├── myapp
│   ├── __init__.py
│   └── workflows
│       ├── __init__.py
│       └── example.py
├── requirements.in
├── requirements.txt
└── version.sh
```

Reply:
its a separate branch